"""Corrigindo erro created

Revision ID: 00bf902eea5a
Revises: 7723ee24cf94
Create Date: 2024-04-26 18:22:35.958082

"""
from typing import Sequence, Union

from alembic import op
import sqlalchemy as sa
from sqlalchemy.dialects import postgresql

# revision identifiers, used by Alembic.
revision: str = '00bf902eea5a'
down_revision: Union[str, None] = '7723ee24cf94'
branch_labels: Union[str, Sequence[str], None] = None
depends_on: Union[str, Sequence[str], None] = None


def upgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('atletas', sa.Column('created_at', sa.DateTime(), nullable=False))
    op.drop_column('atletas', 'creared_at')
    # ### end Alembic commands ###


def downgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('atletas', sa.Column('creared_at', postgresql.TIMESTAMP(), autoincrement=False, nullable=False))
    op.drop_column('atletas', 'created_at')
    # ### end Alembic commands ###
